<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Data Formats &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
<div id="content" role="main" class="columns medium-12 clearfix">
    <section id="ae-help" class="columns medium-9">
        <h2>What format should my raw and processed data be in?</h2>
        <p>Data files are uploaded in zipped archives. Raw files should be
            provided unedited in their native format. Native format is the file
            format that an application, such as the software used by a scanner (e.g.
            gpr files from GenePix) or a sequencing instrument (e.g. BAM format),
            produces. Ideally, you should be able to use the raw unprocessed data
            files from any of the supported software types without having to edit
            the data files in any way. Our checking scripts uses the column headings
            within the file to identify which kind of file it is dealing with, and
            what the quantitation types are. Processed/normalised data can also be
            supplied in native format.</p>

        <p>The following formats are supported:</p>
        <ul>
            <li><a href="#raw" class="subhead">Unprocessed (raw) data
                files</a></li>
            <ul>
                <li><a href="#generic">Generic</a></li>
                <li><a href="#affymetrix">Affymetrix</a></li>
                <li><a href="#genepix">GenePix</a></li>
                <li><a href="#agilent">Agilent</a></li>
                <li><a href="#scanalyze">ScanAlyze</a></li>
                <li><a href="#scanarray">ScanArray</a></li>
                <li><a href="#scanarray">QuantArray</a></li>
                <li><a href="#arrayvision">Arrayvision</a></li>
                <li><a href="#spotfinder">Spotfinder</a></li>
                <li><a href="#bluefuse">BlueFuse</a></li>
                <li><a href="#ucsfspot">UCSF Spot</a></li>
                <li><a href="#nimblescan">NimbleScan</a></li>
                <li><a href="#appliedbiosystems">Applied Biosystems</a></li>
                <li><a href="#codelink">CodeLink</a></li>
                <li><a href="#illumina">Illumina</a></li>
                <li><a href="#imagene">Imagene</a></li>
                <li><a href="#csirospot">CSIRO Spot</a></li>
            </ul>
            <li><a href="#normalized" class="subhead">Normalized data
                files</a></li>
            <ul>
                <li><a href="#generic_norm">Generic</a></li>
                <li><a href="#affynorm">Affymetrix</a></li>
                <li><a href="#illumina">Illumina</a></li>
            </ul>
            <li><a href="#data_matrix" class="subhead">Data Matrix</a></li>
        </ul>

       <h3><a id="raw">Raw (unprocessed) data files</a></h3>
        <p>The following list gives a brief overview of how we recognize
            different file formats. In each case, the data file row containing the
            column headings is identified by matching it to these sets of known
            column headings.</p>

        <h4><a id="generic">Generic</a></h4>
        <p>MetaColumn/MetaRow format files are recognized using the following column headings:</p>

        <table>
            <tr>
                <td>MetaColumn</td>
                <td>MetaRow</td>
                <td>Column</td>
                <td>Row</td>
            </tr>
        </table>

        <h4><a id="affymetrix">Affymetrix</a></h4>
        <p>Our checking scripts recognize and parse CEL and EXP files using
            both the old GDAC formats and the newer GCOS/XDA formats. These file
            formats are detected using the Affymetrix data file parser incorporated
            into the <a href="sourceforge_modules.html">Tab2MAGE package</a>. See
            below for notes on <a href="#affynorm">Affymetrix normalized data
                file formats</a>.</p>

        <h4><a id="genepix">GenePix</a></h4>
        <p>GenePix format files are recognized using the following column
            headings:</p>

        <table>
            <tr>
                <td>Block</td>
                <td>Column</td>
                <td>Row</td>
                <td>X</td>
                <td>Y</td>
            </tr>
        </table>

        <h4><a id="agilent">Agilent</a></h4>
        <p>A file containing these headings is recognized as an Agilent
            format file:</p>

        <table>
            <tr>
                <td>Row</td>
                <td>Col</td>
                <td>PositionX</td>
                <td>PositionY</td>
            </tr>
        </table>

        <h4><a id="scanalyze">ScanAlyze</a></h4>
        <p>The following column headings are recognized as being from a
            ScanAlyze format file:</p>

        <table>
            <tr>
                <td>GRID</td>
                <td>COL</td>
                <td>ROW</td>
                <td>LEFT</td>
                <td>TOP</td>
                <td>RIGHT</td>
                <td>BOT</td>
            </tr>
        </table>

        <h4><a id="scanarray">ScanArray/QuantArray</a></h4>
        <p>ScanArray Express files are recognized from the following
            headings:</p>

        <table>
            <tr>
                <td>Array Column</td>
                <td>Array Row</td>
                <td>Spot Column</td>
                <td>Spot Row</td>
                <td>X</td>
                <td>Y</td>
            </tr>
        </table>

        <p>while the older QuantArray format has these headings:</p>

        <table>
            <tr>
                <td>Array Column</td>
                <td>Array Row</td>
                <td>Column</td>
                <td>Row</td>
            </tr>
        </table>

        <h4><a id="arrayvision">ArrayVision</a></h4>
        <p>The following column headings are recognized as indicating an
            ArrayVision format file:</p>

        <table>
            <tr>
                <td>Primary</td>
                <td>Secondary</td>
            </tr>
        </table>

        <p>Newer "lg2" ArrayVision files are identified by the following
            column headings:</p>

        <table>
            <tr>
                <td>Spot labels</td>
            </tr>
        </table>

        <h4><a id="spotfinder">Spotfinder</a></h4>
        <p>Spotfinder files are recognized by the following column headings:</p>

        <table>
            <tr>
                <td>MC</td>
                <td>MR</td>
                <td>SC</td>
                <td>SR</td>
            </tr>
        </table>

        <h4><a id="bluefuse">BlueFuse</a></h4>
        <p>A file containing the following headings is recognized as a
            BlueFuse file:</p>

        <table>
            <tr>
                <td>COL</td>
                <td>ROW</td>
                <td>SUBGRIDCOL</td>
                <td>SUBGRIDROW</td>
            </tr>
        </table>

        <h4><a id="ucsfspot">UCSF Spot</a></h4>
        <p>UCSF Spot files are recognized by the following column headings:</p>

        <table>
            <tr>
                <td>Arr-colx</td>
                <td>Arr-coly</td>
                <td>Spot-colx</td>
                <td>Spot-coly</td>
            </tr>
        </table>

        <h4><a id="nimblescan">NimbleScan</a></h4>
        <p>NimbleScan files (Feature, Probe and Pair) all contain the
            following headings:</p>

        <table>
            <tr>
                <td>PROBE_ID</td>
                <td>X</td>
                <td>Y</td>
            </tr>
        </table>

        <h4><a id="appliedbiosystems">Applied Biosystems</a></h4>
        <p>Files generated by Applied Biosystems software have the following
            headings:</p>

        <table>
            <tr>
                <td>Probe_ID</td>
                <td>Gene_ID</td>
            </tr>
        </table>

        <h4><a id="codelink">CodeLink</a></h4>
        <p>CodeLink Expression Analysis files are identified using the
            following:</p>

        <table>
            <tr>
                <td>Logical_row</td>
                <td>Logical_col</td>
                <td>Center_X</td>
                <td>Center_Y</td>
            </tr>
        </table>

        <h4><a id="imagene">ImaGene</a></h4>
        <p>ImaGene files are recognized using the following columns:</p>

        <table>
            <tr>
                <td>Meta Column</td>
                <td>Meta Row</td>
                <td>Column</td>
                <td>Row</td>
                <td>Field</td>
                <td>Gene ID</td>
            </tr>
        </table>

        <p>The ImaGene 3.0 format is also supported:</p>

        <table>
            <tr>
                <td>Meta_col</td>
                <td>Meta_row</td>
                <td>Sub_col</td>
                <td>Sub_row</td>
                <td>Name</td>
                <td>Selected</td>
            </tr>
        </table>

        <h4><a id="csirospot">CSIRO Spot</a></h4>
        <p>CSIRO Spot files contain the following columns:</p>

        <table>
            <tr>
                <td>grid_c</td>
                <td>grid_r</td>
                <td>spot_c</td>
                <td>spot_r</td>
                <td>indexs</td>
            </tr>
        </table>
        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h3><a id="normalized">Normalized data files</a></h3>

        <p>Normalized data files may be submitted in any of the above
            formats. In addition, files may be parsed using a number of special
            column headings which can be used to designate a column containing
            reporter or composite sequence identifiers:</p>

        <h4><a id="generic_norm">Generic</a></h4>
        <p>If you have normalized data mapped to the identifiers used in
            your array design, you can simply use a single column containing those
            identifiers. MAGE-TAB supports the use of either Reporter Identifiers or
            CompositeSequence Identifiers for this purpose. Please see these <a
                    href="creating_an_adf.html">ADF help notes</a> for a discussion on these identifier
            types. Thus, either of the following sets of column headers may be used:</p>

        <table>
            <tr>
                <td>Reporter Identifier</td>
                <td>&lt;<i>QT1</i>&gt;</td>
                <td>&lt;<i>QT2</i>&gt;</td>
                <td>&lt;<i>QT3</i>&gt;</td>
            </tr>
        </table>
        <p>
        <table>
            <tr>
                <td>CompositeSequence Identifier</td>
                <td>&lt;<i>QT1</i>&gt;</td>
                <td>&lt;<i>QT2</i>&gt;</td>
                <td>&lt;<i>QT3</i>&gt;</td>
            </tr>
        </table>

        <p>where &lt;<i>QT1</i>&gt;, &lt;<i>QT2</i>&gt; etc. are the names
            of your quantitation types.</p>

        <h4><a id="affynorm">Affymetrix normalized data</a></h4>
        <p>MAGE-TAB recognizes and parses CHP files using both the old GDAC
            formats and the newer GCOS/XDA formats. In addition, Affymetrix data
            normalized by non-Affymetrix methods (e.g. RMA normalization) can be
            parsed. Either CompositeSequence identifiers (see example above) or
            either of the following sets of column headers may be used:</p>

        <table>
            <tr>
                <td>ProbeSet ID</td>
                <td>&lt;<i>QT1</i>&gt;</td>
                <td>&lt;<i>QT2</i>&gt;</td>
                <td>&lt;<i>QT3</i>&gt;</td>
            </tr>
        </table>
        <p>
        <table>
            <tr>
                <td>ProbeSet Name</td>
                <td>&lt;<i>QT1</i>&gt;</td>
                <td>&lt;<i>QT2</i>&gt;</td>
                <td>&lt;<i>QT3</i>&gt;</td>
            </tr>
        </table>

        <p>Again, &lt;<i>QT1</i>&gt;, &lt;<i>QT2</i>&gt; etc. are the names
            of your quantitation types.</p>

        <p><b>Please note: If you have normalised data file(s) that are
            not in one of the formats described above you can still submit this
            data. Ensure you include a 'normalization data transformation protocol'
            in your <a href="creating_an_idf.html" target="_blank">IDF</a>
            describing how these files were created and what the various columns in
            the file(s) represent. Please, if possible, submit the file(s) as a .txt
            file as we cannot process certain file types such as Excel (.xls or
            .xlsx) format.</b></p>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h3><a id="data_matrix">Data Matrix</a></h3>

        <p>If you wish to represent data from more than one assay, scan or
            normalization in a single data file, you will need to reformat it as a
            MAGE-TAB Data Matrix. This is a simplified format which allows data
            columns to be mapped to rows in the SDRF file. The first header line of
            a Data Matrix file describes this mapping, and the second lists the
            quantitation types for each column (e.g. "log2 ratio"). The first column
            is used to map the data rows to identifiers from the array design used.
            Examples are shown here:</p>

        <div align="center">
            <h4>Example non-Affymetrix data matrix</h4>
        </div>

        <img src="nonaffy_datamatrix.png" alt="non Affymetrix data matrix" width=40%
             height=40% />

        <p>In this example, five hybridizations from the 'Assay Name' column
            of the SDRF file <b>[A]</b> are being mapped to log2 ratio values. Each
            row of data is mapped to a Reporter Identifier <b>[B]</b> defined by the
            array design (itself indicated in the <a
                    href="creating_a_sdrf.html#array_design_ref">Array Design REF</a>
            column defined in the SDRF).</p>

        <div align="center">
            <h4>Example Affymetrix data matrix</h4>
        </div>
        <img src="affy_datamatrix.png" alt="Affymetrix data matrix" width=40%
             height=40% />

        <p>In this example, two hybridizations from the 'Assay Name' column
            of the SDRF file <b>[A]</b> are being mapped to data with two different
            quantitation types (CELIntensity, CELStdev). Each row of data is mapped
            to a CompositeElement Identifier <b>[B]</b> defined by the array design.</p>

        <p>There are some limitations imposed by ArrayExpress when
            submitting data in this format. Firstly, each data matrix should
            correspond to assays performed on a single array design. Experiments
            using multiple array designs should use one data matrix per design.
            Secondly, we rely on there being an ordered and regular organisation of
            the columns: first by assay, and then by quantitation type:</p>


        <h4>Correct</h4>
        <table>
            <tr>
                <td>Hybridization REF</td>
                <td style="color:blue">Hyb1</td>
                <td style="color:blue">Hyb1</td>
                <td style="color:red">Hyb2</td>
                <td style="color:red">Hyb2</td>
            </tr>
            <tr>
                <td>Reporter REF</td>
                <td style="color:blue">QT X</td>
                <td style="color:red">QT Y</td>
                <td style="color:blue">QT X</td>
                <td style="color:red">QT Y</td>
            </tr>
        </table>

        <br>

        <h4>Wrong</h4>
        <table>
            <tr>
                <td>Hybridization REF</td>
                <td style="color:blue">Hyb1</td>
                <td style="color:red">Hyb2</td>
                <td style="color:blue">Hyb1</td>
                <td style="color:red">Hyb2</td>
            </tr>
            <tr>
                <td>Reporter REF</td>
                <td style="color:blue">QT X</td>
                <td style="color:blue">QT X</td>
                <td style="color:red">QT Y</td>
                <td style="color:red">QT Y</td>
            </tr>
        </table>


        <p><b>Please note: If you have normalised data file(s) that are
            not in one of the formats described above you can still submit this
            data. Ensure you include a 'normalization data transformation protocol'
            in your <a href="creating_an_idf.html" target="_blank">IDF</a>
            describing how these files were created and what the various columns in
            the file(s) represent. Please, if possible, submit the file(s) as a .txt
            file as we cannot process certain file types such as Excel (.xls or
            .xlsx) format.</b></p>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
        <br>

        <h3><a id="illumina">Illumina</a></h3>
        <p>Several different types of files can be generated from Illumina arrays (see <a
                href="http://www.illumina.com" target="_blank">www.illumina.com</a>).  MAGE-TAB can accept Illumina iDAT files and those produced by Illumina's GenomeStudio.

        <p>Illumina files may contain data from single or multiple assays.
            Data is supported in "Array Data File" or "Derived Array Data Matrix
            File" columns of the MAGE-TAB spreadsheet. The file name should be
            included in all of the rows corresponding to the assays it covers.</p>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
    </section>
    <aside class="columns medium-3 last">
        <h3>Help</h3>
        <ul id="secondary-nav">
            <li><a href="index.html">Help home</a></li>
            <li><a href="quick_start.html">Quick Start guide</a></li>
            <li><a href="how_to_search.html">How to search</a>
                <ul>
                    <li><a href="how_to_search_adf.html">Array design search</a></li>
                    <li><a href="how_to_search_private_data.html">Search for private data</a></li>
                </ul>
            </li>
            <li class="active"><a href="submissions_overview.html">How to submit</a>
                <ul>
                    <li><a href="magetab_spec.html">MAGE-TAB format</a>
                        <ul>
                            <li><a href="creating_an_idf.html">Creating an IDF</a></li>
                            <li><a href="creating_a_sdrf.html">Creating a SDRF</a></li>
                            <li class="active"><a href="data_format.html">Data formats</a></li>
                        </ul>
                    </li>
                    <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                    <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                    <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                    <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
                </ul>
            </li>
            <li><a href="data_availability.html">Learn more about our data</a>
                <ul>
                    <li><a href="data_availability.html">Data access policy</a></li>
                    <li><a href="accession_codes.html">Accession codes</a></li>
                    <li><a href="GEO_data.html">GEO data</a></li>
                </ul>
            </li>
            <li><a href="programmatic_access.html">Other ways to get data</a>
                <ul>
                    <li><a href="programmatic_access.html">Programmatic access</a></li>
                    <li><a href="FTP_archives.html">FTP archives</a></li>
                    <li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
                </ul>
            </li>
            <li><a href="FAQ.html">FAQ</a></li>
            <li><a href="contact_us.html">Contact us</a></li>
        </ul>
    </aside>
</div>
</body>
</html>