<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Submissions Overview &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="grid_24 clearfix">
        <section id="ae-help" class="grid_18 alpha">
            <h2>Experiment submissions overview</h2>
            <span style="color:red">Due to reduced staffing levels between 18 Mar - 8 Apr 2016, curation and brokering of sequencing experiments will take up to 15 working days to complete. If your submission requires urgent processing, please contact us at  <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> and we will raise its priority.  Thank you for your understanding and we apologise for any inconvenience caused.</span>
            <div class="grid_24">
                <div class="calloutbox" style="padding: 20px;  position: relative; background-color: rgb(238, 249, 243);">
                    <h3 class="icon icon-functional" data-icon="_">Annotare webform tool</h3>
                    <br>
                    <a href="/fgpt/annotare_help"><span style="color:blue"><strong>Read this guide before submitting</strong></span></a><br>
                    <a href="/fg/annotare/login"><span style="color:blue"><strong>Annotare login page</strong></span></a>
                    <br><br>
                    <ul>
                        <li><strong>For all non-pipeline submissions of microarray or sequencing experiments.</strong> Pipeline submitters: please contact us at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> and we will advise further</li>
                        <li>Fill in a series of <strong>webforms (auto-saved)</strong>, featuring auto fill-down and copy/paste ("import values") functions</li>
                        <li>Supports up to 1000 assays, no complex design (e.g. one microarray design used, no sample pooling)</li>
                        <li>Upload data files directly within the tool or via <a href="ftp_upload.html">FTP</a></li>
                        <li>Automatic <strong>validation</strong> prior to submission to speed up downstream processing and loading into ArrayExpress</li>
                        <li>Supports <strong>double-blind peer review</strong> in downstream processing, by hiding submitter's contact information from reviewers</li>
                    </ul>
                </div>
            </div>

            <p>&nbsp;</p>
            <p>
                <a href="#WhatData">1. What data sets do we accept?</a><br>
                <a href="#WhatIsASub">2. Major components of an experiment submission</a><br>
            </p>

            <h3><a id="WhatData"></a>What data sets do we accept?</h3>
            <p>ArrayExpress accepts all functional genomics data, e.g. expression profiling, genotyping, chromatin immunoprecipitation, copy-number variation, from microarray and sequencing technologies.</p>
            <p>Microarray submissions follow the "Minimum Information About a Microarray Experiment" (<a href='http://fged.org/projects/miame/'>MIAME</a>) guidelines, while sequencing submissions follow a similar set of guidelines, "Minimum Information About a Sequencing Experiment" (<a href='http://fged.org/projects/minseqe/'>MINSEQE</a>).</p>
            <p><span style="color:red">Please try and get your submission in the best possible shape to avoid <a href='#UpdateCompletedSub'>retrospective fixes post-curation</a>, because such fixes could be tedious and lengthy (especially for <a href='UHTS_submissions.html#HTSDataReleasePolicy'>sequencing experiments</a>).</span></p>
            <p>Data from human samples and individuals that can potentially lead to the identification of the donors (e.g. genomic DNA sequences) can be submitted to ArrayExpress if the data has been consented for public release. Such approvals typically would be given by the relevant ethics committees and <em>ensuring this is the responsibility of the submitters</em>. Identifiable data approved for <a href="/ega/submission/FAQ#DAC">controlled access</a> should be submitted to the <a href="/ega/">European Genome-phenome Archive (EGA)</a> (<a href="uhts_submissions.html#HumanIdentifiableData">more details here</a>).</p>

            <h3><a id="WhatIsASub"></a>Major components of an experiment submission</h3>
            <p>Following the MIAME/MINSEQE guidelines, we ask submitters to provide:</p>
            <div class="grid_1 alpha">
                <p>&nbsp;</p>
            </div>

            <div  class="grid_23 omega">
                <p><strong>Meta-data</strong>: E.g. Background biology of your experiment, aim of the experiment, biological materials/samples used and their characteristics, wet-lab and dry-lab procedures (protocols).</p>
                <p><strong>Raw data files</strong>: These are unprocessed data files obtained from the microarray scanner (e.g. Affymetrix CEL files, Agilent feature extraction files) or from the sequencing machine (e.g. fastq files).</p>
                <ul>
                <li><i>Microarray experiment</i>: Make sure the format of your raw data files matches one of the <a href="/fgpt/annotare_help/accepted_raw_ma_file_formats.html">accepted formats</a>.</li>
                <li><i>Sequencing experiment</i>: Make sure your raw sequence read files are prepared according to <a href="/ena/about/read-file-formats">specifications by the European Nucleotide Archive (ENA)</a> and send them to ArrayExpress by <a href="ftp_upload.html">FTP</a>. You'll also need to gather details about your samples/libraries/data files. For more details, see this <a href = "UHTS_submissions.html">specific help page on sequencing submissions</a>.</li>
                </ul>
                <p><strong>Processed data files</strong>: These are data files generated from the raw data files, often involving e.g. normalisation or background-subtraction. In sequencing experiments, the processing can also involve alignment of sequencing reads to a reference genome, calculation of RPKM/FPKM values, etc.</p>
                <ul>
                <li><i>Microarray experiment</i>: Send us processed data in tab-delimited text (*.txt) files (not Excel spreadsheets) as in this <a href="/fgpt/annotare_help/accepted_processed_ma_file_formats.html">this guide</a>.</li>
                <li><i>Sequencing experiment</i>: We accept tables/spreadsheets of processed data saved as tab-delimited text (*.txt) format. We also accept bam (alignment) files, see <a href="UHTS_submissions.html#BamSpec">bam file specification</a> for more details.</li>
                </ul>
            </div>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
        </section>
        <aside class="grid_6 omega">
            <h3>Help</h3>
    <ul id="secondary-nav">
        <li><a href="index.html">Help home</a></li>
        <li><a href="quick_start.html">Quick Start guide</a></li>
        <li><a href="how_to_search.html">How to search</a>
            <ul>
                <li><a href="how_to_search_adf.html">Array design search</a></li>
                <li><a href="how_to_search_private_data.html">Search for private data</a></li>
            </ul>
        </li>
        <li class="active"><a href="submissions_overview.html">How to submit</a>
            <ul>
                <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
            </ul>
        </li>
        <li><a href="data_availability.html">Learn more about our data</a>
            <ul>
                <li><a href="data_availability.html">Data access policy</a></li>
                <li><a href="accession_codes.html">Accession codes</a></li>
                <li><a href="GEO_data.html">GEO data</a></li>
            </ul>
        </li>
        <li><a href="programmatic_access.html">Other ways to get data</a>
            <ul>
                <li><a href="programmatic_access.html">Programmatic access</a></li>
				<li><a href="FTP_archives.html">FTP archives</a></li>
				<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
            </ul>
        </li>
        <li><a href="FAQ.html">FAQ</a></li>
        <li><a href="contact_us.html">Contact us</a></li>
    </ul>
        </aside>
    </div>
</body>
</html>
