<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Sequencing Submissions &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="grid_24 clearfix">
        <section class="grid_18 alpha">
            <h2>Sequencing Submissions</h2>
            <p>
                <a href="#DataTypes">1. Types of data that can be submitted</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#HumanIdentifiableData"> Potentially identifiable human data</a><br>
                <a href="#HowToSubmit">2. What data files to submit, and how</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#meta_data">Meta-data</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#raw_data_files">Raw data files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#MD5">MD5 checksum of raw files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sending_raw_files">Sending raw files</a><br>
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#processed_data">Processed data files</a><br>
                <a href="#BamSpec">3. BAM file requirements</a><br>
                <a href="#HTSDataReleasePolicy"> 4. Modifying/cancelling a sequencing experiment in ArrayExpress and ENA</a><br>
                
            </p>
            <br>
            <h3><a id="DataTypes"></a>1. Types of data that can be submitted</h3>
            <p>ArrayExpress accepts submissions of functional genomics data generated using high throughput sequencing (HTS) assays like RNA-seq, mostly from <strong>non-human and human non-identifiable</strong> samples. (If you're submitting potentially identifiable human data, please <a href="#HumanIdentifiableData">see below</a>.)</p>
            <p>To submit to ArrayExpress, all you need to do is send us meta-data for your experiment (e.g. experiment description, samples and their attributes, all protocols used) and the raw data files. Submissions without raw data files will not be accepted (see the <a href="#HowToSubmit">submission guide below</a>).</p>
            <p>The meta-data about your experiment will be stored at ArrayExpress, and the raw data files (e.g. fastq files) are eventually stored at the <a href="/ena/about/sra_submissions">Sequence Read Archive</a> (SRA) of the <a href="/ena/">European Nucleotide Archive</a> (ENA). ArrayExpress will transfer the raw data files to the ENA for you so you <i>do not</i> need to submit those files separately to the ENA. You can also send us processed data (i.e. processed from the raw reads, e.g. BAM alignment files, differential expression data, expression values linked to genome coordinates, etc).  Depending on the file format, it will either be stored at ArrayExpress or the ENA.  <span style="color:red">Given the split of meta-data and data files between ArrayExpress and ENA, once your submission is fully processed, it is a lengthy process to modify/update it. Some changes (e.g. cancelling an ENA record which has been released to the public) will not be possible.  Please take a look at our <a href="#HTSDataReleasePolicy">sequencing experiment update/cancellation policy</a> before proceeding.</span></p>
            <p><a id="HumanIdentifiableData"></a><strong>Potentially identifiable human data</strong>:  Data from human samples and individuals that can potentially lead to the identification of the donors (e.g. genomic DNA sequences) can be submitted to ArrayExpress if the data has been consented for public release. Such approvals typically would be given by the relevant ethics committees and <em>ensuring this is the responsibility of the submitters</em>.</p>
            <p>Identifiable data approved for <a href="/ega/submission/FAQ#DAC">controlled access</a> should be submitted to the <a href="/ega/">European Genome-phenome Archive (EGA)</a>. Cases are possible where identifiable data (e.g. raw sequences) are submitted to the EGA, while the related processed data (e.g. RPKM values) are submitted to ArrayExpress, but <em>it is up to the submitter to ensure that such a submission copies to the respective ethics requirements.</em>  To submit processed data to ArrayExpress, please begin by emailing us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> with the EGA study accession number. We will import non-human-identifiable meta-data from EGA in a spreadsheet (which the submitter will have the chance to review), and then match the meta-data with processed data.</p>
            <p>The following diagram summarises the typical data flow:.</p>
            <p><img src="UHTS_HighThroughputSequencingSubmissionDiagram.jpg" alt="Diagram of data submission routes" height="328" width="542"></p>
            <p>Diagram of the submission of different sequencing data types to ArrayExpress or the European Genome-phenome archive.</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
                     
            <h3><a id="HowToSubmit"></a>2. What data files to submit, and how</h3>
            <p><a id="meta_data"></a><strong>Experiment meta-data</strong>: It should be submitted via the <a href="/fg/annotare/login">Annotare webform submission tool</a> (further described on this <a href="submissions_overview.html">submissions overview page</a>).</p>
            
            <p><a id="raw_data_files"></a><strong>Raw data files</strong>: Please prepare them according to <a href='/ena/about/read-file-formats'>ENA specifications</a>. This is a developing field so please do check the specifications every time you submit a new experiment. Data files which do not satisfy ENA's requirements will not be accepted.</p>
            <p>For fastq files, each file must be individually compressed by gzip or bzip2. Do not bundle multiple fastq files into one archive.  For paired-end experiments, if the mate pairs are in two separate files (one file for the forward strand, one for the reverse strand), the two files must be named with the same root and end with extensions such as <code>_1.fq.gz</code> and <code>_2.fq.gz</code>. Examples of naming styles supported by the ENA:            </p>
            <ul>
            <li>sampleA_1.fq.gz / sampleA_2.fq.gz</li>
            <li>sampleA_F.fq.gz / sampleA_R.fq.gz</li>
            </ul>

            
            <p>If you are submitting BAM files as raw data files, please read this important documentation on <a href="#BamSpec">BAM file specification</a>.</p>
            
            <p><a id="MD5"></a><strong>MD5 checksum of raw files</strong>: As sequence files tend to be very large, we implement a file integrity validation step before sending them to the ENA on your behalf. For each raw file, please calculate its MD5 checksum. The checksum is hexadecimal and expressed as a long string of letters and numbers, which looks something like this: <code>eef75461035fb66d9173799d4e26ea97</code>. MD5 checksums are like the files' digital "fingerprints" and it is very unlikely that two non-identical files would generate the same "fingerprint", thus allowing us to verify that each file has not been corrupted during FTP transfer. Remember to calculate the checksum from the actual compressed file (e.g. fastq.gz or fastq.bz2) that is sent to us, not from the uncompressed fastq file or a re-compressed version created at a later time.</p>

            <p>How to calculate MD5 checksums: <a href="http://www.winmd5.com/">Windows user example</a>, <a href="http://www.macappguide.com/2009/04/28/quick-tip-check-md5sum-in-os-x/">Mac user example</a>, <a href="http://heatware.net/linux-unix/how-to-create-md5-checksums-and-validate-a-file-in-linux/">Linux user example</a>.</p> 
            
            <p><a id="sending_raw_files"></a><strong>Sending raw files:</strong> You must send the raw files to ArrayExpress by FTP (see <a href="ftp_upload.html">FTP transfer instructions</a>). Please transfer the compressed files one by one (and not bundling multiple fastq.gz files in one tar.gz archive) to avoid time-out issues and to allow us to process your files promptly.  For <a href="/fg/annotare/login">Annotare</a> to associate the transferred files with your experiment submission, please go to the <code>Samples and Data</code> --> <code>Upload and assign data files</code> section in Annotare, click <code>FTP Upload...</code>, and follow on-screen instructions to fill in file names and their corresponding MD5 checksums.  Annotare will then verify the presence of the files on our FTP site and the MD5 checksums.  If verification passes, you will be able to assign data files to each of your samples.</p>
            
            <p><a id="processed_data"></a><strong>Processed data files</strong>: Please prepare them in tab-delimited text (*.txt) format (not Excel), except for binary files such as bam files. There is no need to compress or zip up the files one by one or as a bundle. Small files (under 10Mb) can be uploaded directly using the <a href="/fg/annotare/login">Annotare</a> submission tool. For any files larger than 10MB, please calculate their MD5 checksums, and send them by <a href="ftp_upload.html">FTP</a>, in the same way as you would for <a href="#sending_raw_files">raw files</a>.</p>
            
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            <p>&nbsp;</p>
            <h3><a id="BamSpec"></a>3. BAM data file requirements</h3>
            <p>If you're submitting raw data files in BAM format, please make sure they satisfy <a href="/ena/about/read-file-formats#bam_format">ENA  specifications</a> as well as the following conditions:</p>
            <ol><li>Each file contains all reads from the sequencing machine, regardless of whether the reads mapped to the reference genome. The reason for this is that we would expect the BAM file to be used to regenerate <i>all</i> the sequencing reads.</li>
            <li>The phred quality score for each base should be included in the file.</li>
            <li>If you have data from paired-end sequencing libraries, for each sequencing run, include data for both mate reads in one single bam file.</li></ol>
            
            <p>If your BAM files contain only mapped reads, then please either create "full" (unfiltered) BAM files, or send us the original read files (e.g. fastq.gz files) as raw data files (again, check <a href='/ena/about/read-file-formats'>ENA specifications</a>).</p>
            <p>BAM files containing only mapped reads can be included in your submission as processed files, as long as they satisfy ENA's specification and that the reference genome used for alignment has been accessioned in the International Nucleotide Sequence Database Collaboration (INSDC, involving <a href="http://www.ddbj.nig.ac.jp">DDBJ</a>, <a href="/ena">ENA</a>, and <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>).</p>
            <p>&nbsp;</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
            
            <h3><a id="HTSDataReleasePolicy"></a>4. Modifying/cancelling a sequencing experiment in ArrayExpress and ENA</h3>
            <p>Please note that with the exception of release date changes and citation updates, we are not able to modify your metadata and/or data files at the ENA once we finish processing your submission, i.e. when the experiment is already loaded into ArrayExpress/ENA. It is therefore important that you get the submission right before you submit.</p>
            <p>If you foresee changes to your experimental metadata or data post submission to ArrayExpress, you may find it simpler to submit directly to ENA. Please see <a href='https://www.ebi.ac.uk/ena/about/submit_and_update'>ENA's submission guide</a> for more details.</p>
            <p><strong>Private experiments</strong>: If your experiment is still private and an update to your ENA record is required, you can re-submit the full metadata and correct files to ArrayExpress. We will keep the same ArrayExpress accession for your data set, but broker this as a brand new submission to ENA for you (which will generate a new ENA Study accession). The ArrayExpress record will then link to the new ENA submission. We will also put you <a href="mailto:datasubs@ebi.ac.uk">in touch with ENA</a> if you wish to cancel the previous, incorrect record at the ENA.</p>
            <p><strong>Public experiments</strong>: It is ENA's policy not to make a public experiment private again or cancel it unless there is an exceptional reason. (See <a href="https://www.ebi.ac.uk/ena/about/data_availability_policy">ENA's Data availability policy</a> for further information.)  Modifying the meta-data and/or data files of a public experiment is very tedious, because they would have been mirrored among other <a href='http://www.insdc.org/'>INSDC</a> partners already (namely, <a href='http://www.ncbi.nlm.nih.gov/genbank'>GenBank</a> and <a href='www.ddbj.nig.ac.jp/'>DDBJ</a>). Please <a href="mailto:datasubs@ebi.ac.uk">contact ENA directly</a> to sort out the existing record at ENA which contains deprecated/incorrect information. If you would like to keep the ArrayExpress and ENA synchronised, please also inform us of any changes which have been agreed and actioned by the ENA, so we can advise further on how to update the ArrayExpress record.</p>           
            <p>&nbsp;</p>
            <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>           

        </section>
        <aside class="grid_6 omega">
            <h3>Help</h3>
    <ul id="secondary-nav">
        <li><a href="index.html">Help home</a></li>
        <li><a href="quick_start.html">Quick Start guide</a></li>
        <li><a href="how_to_search.html">How to search</a>
            <ul>
                <li><a href="how_to_search_adf.html">Array design search</a></li>
                <li><a href="how_to_search_private_data.html">Search for private data</a></li>
            </ul>
        </li>       
        <li><a href="submissions_overview.html">How to submit</a>
            <ul>
                <li class="active"><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
            </ul>
        </li>
        <li><a href="data_availability.html">Learn more about our data</a>
            <ul>
                <li><a href="data_availability.html">Data access policy</a></li>
                <li><a href="accession_codes.html">Accession codes</a></li>
                <li><a href="GEO_data.html">GEO data</a></li>
            </ul>
        </li>
        <li><a href="programmatic_access.html">Other ways to get data</a>
            <ul>
                <li><a href="programmatic_access.html">Programmatic access</a></li>
				<li><a href="FTP_archives.html">FTP archives</a></li>
				<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
            </ul>
        </li>
        <li><a href="FAQ.html">FAQ</a></li>
        <li><a href="contact_us.html">Contact us</a></li>
    </ul>
        </aside>
    </div>
</body></html>
